tests/lib/api_helpers.py

```diff
@@ -0,0 +1,28 @@
+published_cohort_library_get_all_request_body = {
+    "query": (
+        "query PublishedCohortLibraryGetAll { PublishedCohortLibraryGetAll "
+        "{ authors,"
+        "clinicalAtRiskGroupsText,"
+        "id,"
+        "commissioner,"
+        "demographicsText,"
+        "description,"
+        "disclaimerText,"
+        "fixedDateReference,"
+        "name,"
+        "purpose,"
+        "shortName,"
+        "urlSlug,"
+        "summary } }"
+    )
+}
+
+get_by_slug_name_request_body = {
```

Comment:
We should be able to send the urlSlug via the request, and not hardcoded in here